internal/store/postgres/migrations/000059_update_preset_with_delay_and_location.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE preset
+    ADD COLUMN IF NOT EXISTS window_location VARCHAR(10);
+ALTER TABLE preset RENAME COLUMN window_offset TO window_delay;
```

Comment:
While I agree that in our production presets are not actively being managed, but I think we need to alter the value of the existing offset into its negative value. This is just a precaution in case presets are already used.

Reply:
It is not straight to change the offset to delay in all cases. We will have the presents in project config, we can load from there.